### DISCUSSION TITLE
Chinese support for the Search

### DISCUSSION THREAD

**Discussion by remipregh** [squidfunk/mkdocs-material#3031] [General]
Hi @squidfunk,
Does the search feature support the Chinese language as it's now available in the Lunr plugin (https://github.com/MihaiValentin/lunr-languages/blob/master/README.md)?

Thanks

**Comment by squidfunk** [comment]
Unfortunately not, see my comment https://github.com/mkdocs/mkdocs/issues/2509#issuecomment-882689383.

**Reply by remipregh** [reply]
Thanks for the prompt reply!
